bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/tracer/TracersStore.java

```diff
@@ -73,7 +79,7 @@ public Tracer getTracer(String serviceName) {
                 store.put(serviceName, tracer);
             } else {
                 tracer = io.opentelemetry.api.trace.TracerProvider.noop().get("");
-                consoleError.println("error: tracing disabled as tracer provider had not been initialized");
+                consoleError.println("error: tracing disabled as tracer provider had not been initialized.");
```

Comment:
We can add an integration test with the observability disabled to cover these error lines.

Reply:
In order to recreate this scenario, we need to import the `ballerina/observe` module. It's not desirable to import a stdlib module for test cases here. We can add a test to `ballerina/observe` module. Created a task there to add missing negative test cases https://github.com/ballerina-platform/module-ballerina-observe/issues/66.